### ISSUE TITLE
Dead corners still broken

### ISSUE THREAD

**Issue by bcdavasconcelos** [deskflow/deskflow#6767]
Hi, I just bought a Synergy license. Apparently, dead corners is [still broken](https://github.com/symless/synergy-core/issues/3443) on MacOS.

### Operating Systems ###

Server: MacOS 10.15.3
Client: MacOS 10.15.5

### Synergy Version ###

1.12.0-stable-cb8064e8

### Steps to Reproduce ###

1. Install
2. Try to use the feature.
3. Fail.

### Log URLs ###

I can post it, but it seems unnecessary.

* Did restarting Synergy or the computer help? **No.**

**Comment by unformatt** [comment]
Not working for me - server on Mac 10.15.5. Deskflow 1.24.0. I can see the settings are saved in the server conf. They don't seem to have any effect.

**Comment by sithlord48** [comment]
@nbolton  how is this feature expected to work ?

**Comment by nbolton** [comment]
Yeah, the steps are a bit vague. The idea of the feature is that it stops the transition from happening in a particular corner and you can specify the size of the dead area. Useful in scenarios e.g. if you're on Windows and are used to 'throwing' your cursor (i.e. moving with little intended accuracy) into the corner of the screen to hit the X and close a window; less common these days, but used to be a super common habit with smaller screens.
